dashboard/modules/job/job_manager.py

```diff
@@ -756,6 +756,8 @@ def _get_supervisor_runtime_env(
         if env_vars is None:
             env_vars = {}
 
+        env_vars["RAY_worker_niceness"] = "0"
+
```

Comment:
If I remembered correctly, @architkulkarni told me we would probably need to pop this for the driver at `_get_driver_runtime_env` - just like how we deal with `NOSET_CUDA_VISIBLE_DEVICES_ENV_VAR`

Reply:
I think you're right, currently as written this will also set `env_vars["RAY_worker_niceness"] = 0` in the runtime_env of the `ray.init()` in the entrypoint script, so it will be inherited by all tasks and actors.   So we should pop it from `_get_driver_runtime_env`.  (This won't actually "unset" it in the tasks and actors, but if I understand correctly, whatever existing Ray internal code sets it to 15 will now not be overwritten by the runtime_env's "env_vars" field.  This is how it worked for `NOSET_CUDA_VISIBLE_DEVICES_ENV_VAR` at least.)

We should add a quick test for this.  It can just start a simple Ray job with a ray task in its entrypoint script, and then check the env var from inside the task (should be 15) and inside the entrypoint script outside of tasks/actors (should be 0)